work_queue/src/bindings/python3/work_queue.binding.py

```diff
@@ -2073,12 +2075,33 @@ class RemoteTask(Task):
     # @param coprocess  The name of the coprocess which has the function you wish to execute. The coprocess should have a name() method that returns this
     # @param
     # @param command    The shell command line to be exected by the task.
-    # @param kwargs	    keyword arguments used in function to be executed by task. An optional kwarg exec_method can be specified to one of fork, thread, and direct to choose between those three methods of execution
-    def __init__(self, fn, coprocess, **kwargs):
+    # @param args       positional arguments used in function to be executed by task. Can be mixed with kwargs
+    # @param kwargs	    keyword arguments used in function to be executed by task. 
+    def __init__(self, fn, coprocess, *args, **kwargs):
         Task.__init__(self, fn)
-        event = json.dumps(kwargs)
-        Task.specify_buffer(self, event, "infile")
+        self._event = {}
+        self._event["work_queue_kwargs"] = kwargs
```

Comment:
Rename these ones to  `fn_args` and `fn_kwargs`. They relate to the function, not wq.